python/mxnet/test_utils.py

```diff
@@ -2073,6 +2140,22 @@ def compare_optimizer(opt1, opt2, shape, dtype, w_stype='default', g_stype='defa
         compare_ndarray_tuple(state1, state2, rtol=rtol, atol=atol)
     assert_almost_equal(w1.asnumpy(), w2.asnumpy(), rtol=rtol, atol=atol)
 
+
+def compare_symbol_structure(sym1, sym2):
```

Comment:
could be give it a better name? compare is not very clear. Maybe same ops or something like that?